### ISSUE TITLE
Maintain Velocity On Exit

### ISSUE THREAD

**Issue by ThatHackerDudeFromCyberspace** [Lauriethefish/BetterPortals#172]
**What would you like added?**
Is it possible for "Momentum, a function of mass and velocity, is conserved between portals. In layman's terms, speedy thing goes in, speedy thing comes out."
This is mainly for entities like arrows, and also for horizontally placed portals (if it isn't too hard to add, of course)

Also, wondering if a removefromcoords command could be added with the same relative coordinate stuff too?

Thanks

**Is it to solve a particular problem?**
no?

**Comment by Lauriethefish** [comment]
Velocity is already maintained on exit. However, in some cases, an entity is moving so fast that it actually hits the wall on the other side of the origin of a portal before it gets teleported through by the plugin. This causes the velocity to be lost. This is probably avoidable by keeping track of the velocities of nearby entities but it might lead to further bugs.

**Comment by ThatHackerDudeFromCyberspace** [comment]
I tried to jump into a horizontal portal from a tall height, but I didn't fly out of the other end... maybe I did something wrong with the wall actually...
